source/style-guide/writing/write-to-the-user.txt

```diff
@@ -22,26 +28,49 @@ talk to users directly by using *second person*, addressing the user as
       voice.
 
 -  Use second person to avoid the use of gender-specific, third-person
-   pronouns such as *he*, *she*, *his*, and *hers*. If you must use
-   third person, use the pronouns *they* and *their*, but ensure that
-   the noun that the pronoun refers to is plural.
+   pronouns:
+
+   .. hlist::
+      :columns: 4
+
+      - *he*
+      - *she*
+      - *his*
+      - *hers*
+      - *him*
+      - *her*
+      - *himself*
+      - *herself*
+
+   If you must use third person, use the pronouns:
+
+   .. hlist::
+      :columns: 4
+
+      - *they*
+      - *them*
+      - *their*
+      - *themselves*
+
+   Verify that the pronoun's antecedent is plural.
 
 -  Use first-person plural pronouns (*we*, *our*) judiciously. These
-   pronouns emphasize the writer or MongoDB rather than the user, so
-   before you use them, consider whether second person or imperative
-   mood would be more "user friendly."
+   pronouns emphasize the writer or MongoDB rather than the user.
+   Before using first-person pronouns, consider second person or
+   imperative mood instead.
 
    - Use *we recommend* rather than *it's recommended* or *MongoDB
-     recommends*.
+     recommends*. Limit how often you state recommendations. This
+     expresses an opinion on behalf of MongoDB. Make sure that the
+     engineering team or TSE team has expressed this opinion before
+     stating anything as a recommendation.
```

Comment:
Thank you for this!! Spot on!

Just to drive it home, add:

"You should be prepared to defend any recommendations you publish with a ticket number from engineering or the TSE org"